src/prompts/extractJson.ts

```diff
@@ -87,6 +87,13 @@ const prompt = `I have previously sent a text for analysis by GPT-4. The respons
       "unit": "SEK"
     }
   ],
+  "employees": [
+    {
+    "year": "2021", 
+    "value": "12345",
+    "unit": "FTE"
```

Comment:
```suggestion
      "year": "2021", 
      "value": "12345",
      "unit": "FTE"
```

Probably not important, but always good to use consistent indentation to avoid confusion